src/Tribe/Editor.php

```diff
@@ -211,4 +211,35 @@ public function is_classic_editor() {
 
 		return $is_classic_editor_request || $disabled_by_plugin || $disabled_by_filter;
 	}
+
+	/**
+	 * Whether the events are being served using Blocks or the Classical Editor.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool True is using Blocks. False if using the Classical Editor.
+	 */
+	public function is_events_using_blocks() {
+		/**
+		 * Whether the event is being served through blocks
+		 * or the classical editor.
+		 *
+		 * @since TBD
+		 *
+		 * @param bool $is_using_blocks True if using blocks. False if using the classical editor.
+		 */
+		$is_using_blocks = apply_filters( 'tribe_is_using_blocks', null );
+
+		// Early bail: The filter was overridden to return either true or false.
+		if ( is_bool( $is_using_blocks ) ) {
```

Comment:
https://3v4l.org/umO4f

Reply:
While this will work, returning a value like `1` or (string version) `"1"` here would still mean that the filtered value is true.

WordPress will, typically, check if the early bail filter value is not `null`

```suggestion
		if ( null !== $is_using_blocks ) {
```

Any other value would be cast to true and returned; I would modify the logic to that.